### PR TITLE
Fix walking speed bug. Added more travel options.

### PR DESCRIPTION
**Pull request by h4v1nfun**
Fix walking speed bug. Added more travel options, truck, car and plane

### CODE REVIEW

**Comment by h4v1nfun** [comment]
more travel options

Fixed walking speed bug, added truck @ 80km/h, car @ 120km/h and plane @ 300km/h

**Comment by d3viant0ne** [comment]
@h4v1nfun - 10km/h is too fast to hatch eggs reliably. 9km/h seems to be the sweet spot.

**Comment by h4v1nfun** [comment]
@d3viant0ne fixed 👍

**Comment by AbelIngrand** [comment]
Perhaps replace Truck with Bike from https://github.com/iam4x/pokemongo-webspoof/pull/129 ? I guess Plane is too fast and could provoke soft bans. Have you tested it ?

**Comment by d3viant0ne** [comment]
I have to agree with @AbelIngrand here. If you want to move large distances rapidly, it's best to plan for that location and than have the entire spoofing system running at that location when you launch to avoid getting slapped with a ban.

Moving 300 km/h is going to get you nailed imo as there is no cellular signal at altitude and play wifi doesn't provide geolocation.

**Comment by h4v1nfun** [comment]
@AbelIngrand just wandering the same, perhaps scrap plane and implement #129 bike and leave truck

**Comment by AbelIngrand** [comment]
Yes good idea, keeping truck and implementing bike.

Bike seems like a good speed to move around dense zones even though it doesn't allow hatching

**Comment by h4v1nfun** [comment]
Done, screenshot updated.

**Comment by d3viant0ne** [comment]
LGTM

//cc @iam4x

File: src/models/autopilot.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Autopilot {
   @observable paused = false
   @observable running = false // is the autopilot running
   @observable steps = []
-  @observable speed = 12 / 3600 // 0.0033 ~= 3,3m/s ~= 12 km/h
+  @observable speed = 9 / 3600 // 0.0033 ~= 3,3m/s ~= 12 km/h
```
Comment (iam4x) [inline src/models/autopilot.js]: update the comment as well so 👍

**Comment by iam4x** [comment]
Thank's for the PR @h4v1nfun and see you in Lisboa, coming back to portugal next week. Love this city <3

File: src/models/autopilot.js (hunk 1 above)
Comment (h4v1nfun) [inline src/models/autopilot.js]: done :) hope you have a great time here in Portugal 👍

**Comment by iam4x** [comment]
🚢

**Comment by d3viant0ne** [comment]
Cheers @h4v1nfun

**Pull request merged**